### DISCUSSION TITLE
Collect all /prewards in one click (for daily rewards)

### DISCUSSION THREAD

**Discussion by Vehzx** [Zrips/CMI#3351] [Feature Requests]
**Description of feature/improvement request:**
A way to claim all prewards from chat, maybe a "Shift+Click" to claim all feature, with chat spam restrictions players can only click claim button every 2 seconds, when there is alot of built up /prewards players get annoyed at the time it takes.
---
### Detailed explanation of what it should do, when and in what situation: ###
Simply Shift+Click on the "Claim" message when doing /prewards
to claim all the rewards available.
### Basic explanation as to why this feature is needed and/or useful to have: ###
So players can claim all their rewards with ease without having to click multiple times over and over.

PS - I did look to see if this was a feature but from what I could find I didn't see anything about it, if i'm wrong let me know, thanks.

**Comment by Xuwznln** [comment]
Also need this. I simply have no idea what plugin is anti spam command and if a player click too many times to claim his prewards, he got kicked. So a one click claim all prewards will be helpful.
